### PR TITLE
Switch to the SeaCat G3

### PR DESCRIPTION
**Pull request by ateska**

### CODE REVIEW

File: zScanner/Coordinators/AppCoordinator.swift [hunk 1]
```diff
@@ -89,7 +94,6 @@ extension AppCoordinator: LoginFlowDelegate {
 extension AppCoordinator: DocumentsFlowDelegate {
     func logout() {
         removeUserSession()
-        SeaCatClient.reset()
```
Comment (jakubskorepapoq) [inline zScanner/Coordinators/AppCoordinator.swift]: How does the logout work now? I don't see any call to /logout endpoint

Comment (skorepak) [inline zScanner/Coordinators/AppCoordinator.swift:92]: @ateska 
How does the login work now? I don't see any call to `/logout` endpoint

File: zScanner/Services/Network/Medicalc/Requests/LoginRequest.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+struct LoginRequest: Request {
+    typealias DataType = EmptyResponse
+    
+    var endpoint: Endpoint = SeaCatPKIEndpoint.login
+    var method: HTTPMethod = .post
+    var parameters: Parameters? = nil
+    var headers: HTTPHeaders = [:]
+    
+    init(username: String, password: String) {
+        //This has to go in the body!!! of the POST request
```
Comment (skorepak) [inline zScanner/Services/Network/Medicalc/Requests/LoginRequest.swift:20]: Conform `LoginRequest` to the protocol `ParametersJsonEncoded `

Comment (ateska) [reply zScanner/Services/Network/Medicalc/Requests/LoginRequest.swift:20]: A sample code is welcomed ;-)

File: zScanner/Coordinators/AppCoordinator.swift (hunk 1 above)
Comment (ateska) [reply zScanner/Coordinators/AppCoordinator.swift:92]: SeaCat identity is no longer tightly connected with a logged user. So SeaCat reset is not truly needed here ... or maybe yes, but that will be seen later.

**Review by skorepak — APPROVED** [review]

**Pull request merged**